libs/architectures/mlpe/architectures/flows/flow.py

```diff
@@ -32,10 +50,16 @@ def linear_block(self) -> transforms.Transform:
     def distribution(self) -> distributions.Distribution:
         pass
 
+    @property
+    def context_dim(self):
```

Comment:
let's keep `sample_tensor` as an optional argument with a default of what you have. Sometimes it is useful for debugging

Reply:
Good idea

Reply:
Actually on second thought, if someone needs to debug, they can simply instantiate the `embedding` and pass tensors through there. I think this should stay as a `property` of the instantiated flow.